index.d.ts

```diff
@@ -41,7 +41,7 @@ interface SendOptions {
   etag?: boolean;
   extensions?: string[];
   immutable?: boolean;
-  index?: string[];
+  index?: string[] | boolean;
```

Comment:
```suggestion
  index?: string[] | false;
```

Reply:
It should be `false` only. 
https://github.com/pillarjs/send/blob/de073ed3237ade9ff71c61673a34474b30e5d45b/index.js#L1056